livetest/citation_test.py

```diff
@@ -188,5 +189,42 @@ def test_on_github_but_repo_doesnt_exist(self):
         self.assertTrue('Error requesting file: ' in str(context.exception))
 
 
+class CitationTestOverride(unittest.TestCase):
+
+    def setUp(self):
+        # with override
+        url = "https://github.com/citation-file-format/cff-converter-python"
+        override = yaml.safe_load('doi: "thebestdoi.23678237"\ndate-released: 2018-03-05')
```

Comment:
Why are you loading yaml string? Why are you not using a dict directly?

Reply:
That's what I did initially but I couln;t get it to work. Maybe I should give it another try.

Reply:
Specifically, the date was a problem, because it should not be a string but a date object. I've now changed it.